utils/secrets.py

```diff
@@ -0,0 +1,10 @@
+from passlib.context import CryptContext
+
+
+
+
+SECRET_KEY = "FMe1o0baNLQ_ntPVuK2FTGWwxc_m1KfuKWp0xgReaJg"
```

Comment:
secretes should be in environment variables
read https://fastapi.tiangolo.com/advanced/settings/